doc/nrf/ug_matter_creating_accessory.rst

```diff
@@ -122,8 +123,8 @@ Complete the steps in the following subsections to modify the main loop.
 Edit the event queue
 ====================
 
-The main application loop is based on a queue on which the events are posted by ZCL callbacks, the application itself or by other entities, such as Zephyr timers.
-In each iteration, the event is dequeued and a corresponding event handler is called.
+The main application loop is based on a queue on which events are posted by ZCL callbacks and other application components, such as Zephyr timers.
```

Comment:
```suggestion
The main application loop is based both on a queue on which events are posted by ZCL callbacks and on other application components, such as Zephyr timers.
```

Reply:
I meant that events can be posted by both ZCL callbacks and other components.